packages/hdwallet-ledger/src/bitcoin.ts

```diff
@@ -72,13 +72,13 @@ export async function btcGetPublicKeys(
       payload: { publicKey: parentPublicKeyHex },
     } = res1;
     const parentPublicKey = compressPublicKey(Buffer.from(parentPublicKeyHex, "hex"));
-    const parentFingerprint = new DataView(bitcoin.crypto.ripemd160(bitcoin.crypto.sha256(parentPublicKey))).getUint32(0);
+    const parentFingerprint = new DataView(bitcoin.crypto.ripemd160(bitcoin.crypto.sha256(parentPublicKey)).buffer).getUint32(0);
```

Comment:
Buffer can return unexpected data because of shared heap but since you're getting the first value, it might be okay? @reidrankin ?

Reply:
The `DataView` constructor was not happy with its first argument being a plain `Buffer`. It wants an `ArrayBufferLike`